index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <!-- code-review -->
+  <body>
+    <main>
+      <div class="heading">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+      </div>
+      <div class="form">
+        <form id="survey-form">
+          <label id="name-label" for="name">Name</label>
+          <input type="text" id="name" placeholder="Enter your name" required>
```

Comment:
- [ ] Make sure the input matches the template's design. Use proper padding for each input.